src/output-json-ike.c

```diff
@@ -46,12 +47,14 @@
 
 #include "app-layer-ike.h"
 #include "output-json-ike.h"
-
```

Comment:
nit

Reply:
An you want that newline _put back in_. OK.

Reply:
Yes, just minimize the changes :-)

Reply:
Done